clang/test/CodeGenSYCL/check-direct-attribute-propagation.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clang_cc1 -fno-sycl-force-inline-kernel-lambda -fsycl-is-device -internal-isystem %S/Inputs -triple spir64-unknown-unknown -disable-llvm-passes -sycl-std=2020 -opaque-pointers -emit-llvm -o - %s | FileCheck %s
+// RUN: %clang_cc1 -O2 -fno-sycl-force-inline-kernel-lambda -fsycl-is-device -internal-isystem %S/Inputs -triple spir64-unknown-unknown -disable-llvm-passes -sycl-std=2020 -opaque-pointers -emit-llvm -o - %s | FileCheck %s
```

Comment:
Little confused here. What is the optimization level here now if you don't pass -O2? What was it before?

Reply:
That's right. The default level is set by `getOptimizationLevel` function from `clang/lib/Frontend/CompilerInvocation.cpp`.
With this change it's `llvm::CodeGenOpt::None` (O0) by default (was `llvm::CodeGenOpt::Default` (O2) before).